core/templates/pages/exploration-player-page/learner-experience/post-chapter-recommendations.component.html

```diff
@@ -1,64 +1,105 @@
 <div class="post-chapter-recommendations-container">
-  <p class="post-chapter-recommendation-text">Here's what you could do next!</p>
+  <p class="post-chapter-recommendation-text">{{ 'I18N_END_EXPLORATION_RECOMMENDATION_MESSAGE' | translate }}</p>
   <div class="post-chapter-recommendation-cards-container">
-    <div class="recommendation-card">
-      <svg width="192" height="150" class="recommendation-card-svg">
-        <a [attr.href]="nextLessonLink">
-          <rect width="192"
+    <div *ngIf="nextStoryNodeTitle" class="recommendation-card-wrapper">
+      <div class="recommendation-card">
+        <svg width="192" height="150" class="recommendation-card-svg">
+          <title>Next story node card image</title>
```

Comment:
It may not be clear to the user what a "node card" refers to. Can we rename it such that it is clearer? Note that screen readers will announce SVG titles.

Reply:
Yes, I actually added the title tags to allows screen readers to read out the recommendation cards -- you're right, I've renamed it to 'chapter' instead of 'story node'.